app/src/main/java/eu/kanade/tachiyomi/data/library/LibraryUpdateService.kt

```diff
@@ -604,10 +612,12 @@ class LibraryUpdateService(
             notifications.add(Pair(notification(Notifications.CHANNEL_NEW_CHAPTERS) {
                 setSmallIcon(R.drawable.ic_tachi)
                 try {
-                    val icon = GlideApp.with(this@LibraryUpdateService)
-                        .asBitmap().load(manga).dontTransform().centerCrop().circleCrop()
-                        .override(256, 256).submit().get()
-                    setLargeIcon(icon)
+
+                    val request = LoadRequest.Builder(this@LibraryUpdateService).data(manga)
+                    .transformations(CircleCropTransformation()).size(width = 256, height = 256)
+                        .target { drawable -> setLargeIcon((drawable as BitmapDrawable).bitmap) }.build()
+
+                    Coil.imageLoader(this@LibraryUpdateService).execute(request)
```

Comment:
So for whatever reason, old covers made by the old cache system do not load instantly.

Checking this out for a bit adding `.await()` at the end of `Coil.imageLoader(this@LibraryUpdateService).execute(request)` solves this issue, this method just has to be changed to a suspend fun.

Also from the quick readup I've done, adding `.networkCachePolicy(CachePolicy.DISABLED)` to the request will make sure it never fetches from the network. This shouldn't matter here since we haven't updated the manga yet, but if you've already updated the url in Neko by the time this runs, then the await will freeze this until the network call finishes.